File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.java
import java.util.Scanner;

public class Main {
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		int su = sc.nextInt();
		int su1 = su;
		int co=0;
		while (true) {
			while(true) {
				int a=su1/10;
				int b = su1%10;
				int c= (a+b)%10;
				su1 = b*10+c;
				co++;
				break;
			}
			if(su1==su) {
				break;
			}
		}
		System.out.println((co));
	}
}

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
su = int(input())
su1=su
co=0
while True:
    while True:
        a=su1//10
        b=su1%10
        c=(a+b)%10
        su1=b*10+c
        co+=1
        break
    if  su1==su :
        break
print(co)